pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -384,6 +382,14 @@ func (r *Builder) mapCPU(vm *model.Workload, object *cnv.VirtualMachineSpec) {
 	} else if r.Plan.Spec.PreserveClusterCPUModel {
 		r.setCpuFlags(r.getClusterCpu(vm), object)
 	}
+	if object.Instancetype != nil {
+		return
+	}
```

Comment:
ok, here it's a bit more tricky since PreserveClusterCPUModel is related to CPU, so here just to replace the early return

Reply:
Same, `DedicatedCPUPlacement` above collides with instancetype.

Reply:
yeah i saw. also hugepages and such for memory. i will leave them clean (nothing in CPU or memory will be set). thanks @0xFelix